connector/dao/src/main/java/it/pagopa/selfcare/mscore/connector/dao/model/mapper/PecNotificationEntityMapper.java

```diff
@@ -0,0 +1,18 @@
+package it.pagopa.selfcare.mscore.connector.dao.model.mapper;
+
+import it.pagopa.selfcare.mscore.connector.dao.model.PecNotificationEntity;
+import it.pagopa.selfcare.mscore.model.pecnotification.PecNotification;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+import java.util.UUID;
+
+@Mapper(componentModel = "spring", imports = UUID.class)
+public interface PecNotificationEntityMapper {
+
+    @Mapping(target = "id", defaultExpression = "java(UUID.randomUUID().toString())")
+    @Mapping(target = "institutionId", defaultExpression = "java(UUID.randomUUID().toString())")
```

Comment:
we should always have an institutionId....random value with no reference to an institution is not accepted